mne_icalabel/datasets/icalabel/icalabel.py

```diff
@@ -57,13 +62,4 @@ def data_path(path=None, force_update=False, update_path=True, download=True, ve
         download=download,
         processor=pooch.Unzip(extract_dir=f"./{folder_name}"),
     )
-    dpath = str(dpath)
-
-    # Do some wrangling to deal with nested directories
-    bad_name = os.path.join(dpath, "mne-testing-icalabel-data-main")
-    if os.path.isdir(bad_name):
-        os.rename(bad_name, dpath + ".true")
-        shutil.rmtree(dpath)
-        os.rename(dpath + ".true", dpath)
-
```

Comment:
@adam2392 Why is this required? The `if` statement is not tested, and locally I can't figure out a case in which I get into it.

Reply:
Nevermind.